dotCMS/src/test/java/com/dotmarketing/business/cache/provider/h22/H22CacheTest.java

```diff
@@ -32,65 +34,88 @@
 					+ "C"
 					+ "ASDSADFDDSFasfddsadasdsadsadasdq3r4efwqrrqwerqewrqewrqreqwreqwrqewrqwerqewrqewrqwerqwerqwerqewr43545324542354235243524354325423qwerewds fds fds gf eqgq ewg qeg qe wg egw	ww	eeR ASdsadsadadsadsadsadaqrewq43223t14@#$#@^%$%&%#$sfwf	erqwfewfqewfgqewdsfqewtr243fq43f4q444fa4ferfrearge");;
 	final String CANT_CACHE_KEYNAME = "CantCacheMe";
-	final int numberOfPuts = 5000;
+	final int numberOfPuts = 1000;
```

Comment:
Codacy found an issue: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5918493)